nearlib/README.md

```diff
@@ -1,22 +1,90 @@
-Near lib
-====
+# Near lib
 
 Near lib is a javascript library for development of dapps on near.
 
 ## Install dependencies
-
 ```
 npm install
 ```
-
 ## Test
 
 Start the node by following instructions from nearcore/README.md, then
-
 ```
 npm test
 ```
-
 ## Integration test
 
-Contract "hello.wasm" source code location: https://studio.nearprotocol.com/?f=mfaccXt
+Contract "hello.wasm" source code location: <https://studio.nearprotocol.com/?f=mfaccXt>
```

Comment:
that link is likely outdated

Just link https://github.com/nearprotocol/nearcore/tree/master/tests/hello instead?

Reply:
done